es/VALUES.md

```diff
@@ -1 +1,21 @@
-placeholder
\ No newline at end of file
+### FolksWhoCode Community Values
+
+**FolksWhoCode** tiene como objetivo crear y promover un espacio diverso e inclusivo
+dentro de todos sus proyectos, así como en cualquier espacio en el que los
+administradores y usuarios de la organización tengan voz.
+
+Todos nuestros proyectos son `beginner-friendly` por defecto, por eso parte de
+nuestro trabajo consiste en crear un ambiente cómodo y seguro para todos los
+nuevos *contributors* que lleguen a nuestros repos. Nuestra guía de contribución
+explica esto detalladamente.
+
+Admin/senior/privileged users will act as mentors/sponsors of newcomers and will
```

Comment:
It looks like you forgot to tanslate those last paragraphs. Thank you for taking the time to translate this! Maybe you can fix this and we'll get it merged! :)

Reply:
Ok! 👍